type-morph/main.ts

```
@@ -171,6 +183,45 @@ function parseImportLine(line: string, language: string): { module: string; item
         return { module: starMatch[2], items: [starMatch[1]] };
       }
       break;
+    case 'java':
+    case 'kotlin':
+      const langMatch = line.match(/^import\s+([^;]+);$/);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)